biodatasets/epicqa/epicqa.py

```diff
@@ -0,0 +1,295 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import json
+
+import datasets
+
+from utils import schemas
+from utils.constants import Tasks
+from utils.configs import BigBioConfig
+
+_CITATION = """\
+Unknown
+"""
+
+_DESCRIPTION = """\
+In response to the COVID-19 pandemic, the Epidemic Question
+Answering (EPIC-QA) track challenges teams to develop systems
+capable of automatically answering ad-hoc questions about the
+disease COVID-19, its causal virus SARS-CoV-2, related corona
+viruses, and the recommended response to the pandemic.
+"""
+
+_DATASETNAME = "EpicQA"
+
+_HOMEPAGE = "https://bionlp.nlm.nih.gov/epic_qa/#collection"
+
+_LICENSE = "DUA-NC"
+
+_BASE = "https://bionlp.nlm.nih.gov/epic_qa/data/epic_qa_"
+
+_URLs = {
+    "epicqa_research_articles_source": _BASE + "cord_2020-10-22-split-corrected.tar.gz",
+    "epicqa_research_articles_bigbio_qa": _BASE + "cord_2020-10-22-split-corrected.tar.gz",
+    
+    "epicqa_consumer_articles_source": _BASE + "consumer_2020-11-02.tar.gz",
+    "epicqa_consumer_articles_bigbio_qa": _BASE + "consumer_2020-11-02.tar.gz",
+}
+
+_DIRS = {
+    "research_articles" : [
+        "2020-10-22-new/",
+    ],
+    "consumer_articles" : [
+        "epic_qa_consumer_2020_11-02/ask_science-2020-10-29/",
+        "epic_qa_consumer_2020_11-02/ccns-trec/",
+        "epic_qa_consumer_2020_11-02/chqa-2020-10-09/",
+    ],
+}
+
+_SUPPORTED_TASKS = [Tasks.QUESTION_ANSWERING]
+_SOURCE_VERSION = "1.0.0"
+_BIGBIO_VERSION = "1.0.0"
+
+class EpicQaDataset(datasets.GeneratorBasedBuilder):
+    """EpicQA"""
+
+    DEFAULT_CONFIG_NAME = "source"
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    BIGBIO_VERSION = datasets.Version(_BIGBIO_VERSION)
+
+    BUILDER_CONFIGS = [
+
+        BigBioConfig(
+            name="epicqa_research_articles_source",
+            version=SOURCE_VERSION,
+            description="EpicQA source schema for the Research Articles subset",
+            schema="source",
+            subset_id="epicqa_research_articles",
+        ),
+
+        BigBioConfig(
+            name="epicqa_consumer_articles_source",
+            version=SOURCE_VERSION,
+            description="EpicQA source schema for the Consumer Articles subset",
+            schema="source",
+            subset_id="epicqa_consumer_articles",
+        ),
+
+        BigBioConfig(
+            name="epicqa_research_articles_bigbio_qa",
+            version=BIGBIO_VERSION,
+            description="EpicQA BigBio schema for the Research Articles subset",
+            schema="bigbio_qa",
+            subset_id="epicqa_research_articles",
+        ),
+
+        BigBioConfig(
+            name="epicqa_consumer_articles_bigbio_qa",
+            version=BIGBIO_VERSION,
+            description="EpicQA BigBio schema for the Consumer Articles subset",
+            schema="bigbio_qa",
+            subset_id="epicqa_consumer_articles",
```

Comment:
totally up to you, but it might make things a bit less verbose if we removed `articles` from all `name` and `subset_id` fields